EventFilter/EcalTBRawToDigi/src/CamacTBDataFormatter.cc

```diff
@@ -168,7 +166,7 @@ void CamacTBDataFormatter::interpretRawData( const FEDRawData & fedData,
   LogDebug("CamacTBDataFormatter") << "event number in spill: "<< b;
 
   a = buffer[wordCounter];wordCounter++;
-  b = (a& 0xffffff);
+  b = (a& 0xffff);
```

Comment:
That was a bad copy/paste error from me, i apologize. I have fixed it and
I'm rechecking everything else just to be sure. Thanks for pointing it out
Carl.

On Wed, Jun 8, 2016 at 12:51 AM, Carl Vuosalo <notifications@github.com>
wrote:

> In EventFilter/EcalTBRawToDigi/src/CamacTBDataFormatter.cc
> <https://github.com/cms-sw/cmssw/pull/14220#discussion_r66166838>:
>
> > @@ -168,7 +166,7 @@ void CamacTBDataFormatter::interpretRawData( const FEDRawData & fedData,
> >    LogDebug("CamacTBDataFormatter") << "event number in spill: "<< b;
> >
> >    a = buffer[wordCounter];wordCounter++;
> > -  b = (a& 0xffffff);
> > +  b = (a& 0xffff);
>
> Why did 0xffffff get changed to 0xffff? Such a change could zero out the
> high bits of b, which could affect run-time behavior.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/14220/files/33fbf3cb0e98aa31d54b735bb7e79e988904d8a8#r66166838>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACN6mSzdnVqLkS_crfhjg7cFU5dPFpz4ks5qJfX8gaJpZM4IOSjp>
> .
>